### PR TITLE
#930 Add some print stylesheets

### PR DESCRIPTION
**Pull request by willemstuursma**
This adds some print stylesheets that allow for the generation of nice PDF files.

### CODE REVIEW

File: source/theme/styles/components/_print.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
+@media print {
+
+  body {
+    padding: 1.5em 1em;
```
Comment (vernondegoede) [inline source/theme/styles/components/_print.scss:4]: I see a mix of px & em. Can we use pixels?

File: source/theme/styles/components/_print.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
+@media print {
+
```
Comment (vernondegoede) [inline source/theme/styles/components/_print.scss:2]:
```suggestion
```

File: source/theme/styles/components/_print.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    padding: 1.5em 1em;
+  }
+
+  .main-navigation {
+    position: relative;
+  }
+
+  .product-navigation { display: none; }
+
+
```
Comment (vernondegoede) [inline source/theme/styles/components/_print.scss:13]:
```suggestion
```

File: source/theme/styles/components/_print.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    overflow: hidden;
+  }
+
+  .parameter__children-button {
+    display: none;
+  }
+
+  .parameter__children {
+    position: relative;
+    top: 0;
```
Comment (vernondegoede) [inline source/theme/styles/components/_print.scss:48]: Do we need `top` when using `relative`?

File: source/theme/styles/components/_print.scss [hunk 5]
```diff
@@ -0,0 +1,57 @@
+@media print {
```
Comment (vernondegoede) [inline source/theme/styles/components/_print.scss:1]: Can you please include a screenshot of the result in this PR?

File: source/theme/styles/components/_print.scss (hunk 4 above)
Comment (willemstuursma) [reply source/theme/styles/components/_print.scss:48]: I copied this from the `active` class that is assigned to this element when you click the "Show child parameters" link. I don't know anything about CSS. If you think it should be removed I will remove it.

File: source/theme/styles/components/_print.scss (hunk 1 above)
Comment (willemstuursma) [reply source/theme/styles/components/_print.scss:4]: Sure how many pixels should I pick, just something that looks good?

Comment (lucasfeliciano) [reply source/theme/styles/components/_print.scss:4]: @willemstuursma you can convert em to pixels.

```
1 em   = 16px
1.5em = 24px
```

```suggestion
    padding: 24px 16px;
```

Comment (lucasfeliciano) [reply source/theme/styles/components/_print.scss:4]: @willemstuursma you can convert em to pixels.

```
1 em   = 16px
1.5em = 24px
```

```suggestion
    padding: 24px 16px;
```

**Comment by willemstuursma** [comment]
@vernondegoede please let me know if any further changes are needed.

**Review by mollierobbert — APPROVED** [review]

**Pull request merged**